docs/errors.md

```diff
@@ -167,6 +163,14 @@ To fix this warning, you may either use [import maps](/docs/import-maps.md) or [
 </script>
 ```
 
+# SystemJS Warnings
+
+This sections lists the SystemJS warnings that you may encounter.
+
+## W1
+
```

Comment:
Does the warning have a title? We should try to keep the titles in sync with the messages for both errors and warnings. No need to do all in this PR, but at least the one we are doing now as a habit.

Reply:
👍 good catch